tika-core/src/test/java/org/apache/tika/TikaTest.java

```diff
@@ -569,10 +571,12 @@ protected String getText(String filePath, Parser parser, Metadata metadata,
     public String getText(InputStream is, Parser parser, ParseContext context, Metadata metadata)
             throws Exception {
         ContentHandler handler = new BodyContentHandler(1000000);
-        try {
+        try(is){
```

Comment:
This is what was causing the checkstyle problems. Fixing locally now. Will push with fix.